xlive/Blam/Engine/Networking/Session/NetworkSession.h

```diff
@@ -45,7 +45,7 @@ struct s_membership_player;
 struct s_membership_peer;
 struct s_session_observer_channel;
 
-#define NETWORK_SESSION_PEERS_MAX (16 + 1)
+#define K_NETWORK_SESSION_NUMBER_OF_PEERS (16 + 1)
```

Comment:
Took a look, k_network_maximum_machines_per_session is the define used for peers